### ISSUE TITLE
Question: Does it work on phones?

### ISSUE THREAD

**Issue by QuinnOSS** [JeffZ87/Pokemon-Planet-Farmer#1]
Hi this project seems interesting, but what if or will a player get ban for using this tool.

**Comment by QuinnOSS** [comment]
Oh wait nevermind so this project actually reads the screen and helps to move to character for you. Ok understand. But how does the tool actually know what its battling. Does it have an AI that will read the screen and make a decision, or does it interact with the server somehow

**Comment by QuinnOSS** [comment]
The reason why im asking these stuff is because if it directly interacts with da server. then the player will get ban. I know this cuz I tried to make a bot that talks directly to the server to grind faster and I got banned immediately.

**Comment by QuinnOSS** [comment]
I see that u have like a driver or something to recognize what is on screen. But is it accurate and how does it work? me just interested cuz my programming sucks and da code does not really seem to actually work. My pc also got taken away, so I got nothing to do programming on. I just got my small phone

**Comment by JeffZ87** [comment]
Hi. 

To your first question, I wrote this program just for fun. So, use it at your own risk. 

To your second comment, the program I wrote is anticipated to run on Windows (maybe on Mac with some configuration). For my program to work correctly, you must install Tesseract-OCR on your machine and link path to the executable, which you cannot do on phones. 

If you are interested in programming, definitely look into creating your own projects! Best of Luck :)
